custom/icds_reports/templates/icds_reports/icds_app/mobile/map-chart.html

```diff
@@ -22,8 +22,10 @@
         <div class="col-xs-6 section-heading" ng-bind="step.label" ng-click="$ctrl.goToRoute(step.route)" ng-repeat="step in $ctrl.steps"></div>
       </div>
     </div>
-    <div ng-if="$ctrl.step === 'map'">
-      I'm a Map!
+    <div ng-if="$ctrl.step === 'map'" style="padding-top: 10px">
+      <div class="chartView col-xs-11">
+        I'm a Map!
+      </div>
```

Comment:
this just adds the white box below:

![image](https://user-images.githubusercontent.com/66555/70306635-0a535c00-1810-11ea-8957-19b922799500.png)

Anyways this part will be totally redone once we have maps